### DISCUSSION TITLE
How should Nest.js integration look like in v5 after beforeExit hook removal?

### DISCUSSION THREAD

**Discussion by mykhailo-monchak** [prisma/prisma#20173] [Q&A] [answered]
Hey there,

Since v5 we do not have `beforeExit` hook anymore. But it was essential in terms of [Nest.js integration](https://docs.nestjs.com/recipes/prisma).

How should the configuration be changed since the update?

**Comment by nurul3101** [comment, marked as answer]
Hello @mikhail-monchak 👋 

We have a open GitHub Issue for the same:
- https://github.com/prisma/prisma/issues/20171

We do plan to update the Nest JS guide with the suggested solution for Prisma 5.0 and above ASAP.

**Reply by nurul3101** [reply]
@mikhail-monchak There's an update in which you might be interested:
- https://github.com/prisma/prisma/issues/20171#issuecomment-1632599621

**Reply by nurul3101** [reply]
I'll mark this as resolved.
We have updated Nest Js Documentation with recommended steps.
Here's the merged PR:
- https://github.com/nestjs/docs.nestjs.com/pull/2797
